### ISSUE TITLE
New Helm Chart available

### ISSUE THREAD

**Issue by sspreitzer** [TremoloSecurity/kube-oidc-proxy#32]
Hi there 👋 

I just create a Helm Chart based on your work. Documented and published for public use. Maybe you can give it a push.

* https://github.com/sspreitzer/helm-kube-oidc-proxy

**Comment by mlbiam** [comment]
looks very cool!  i'll add a link to it in the docs

**Comment by mlbiam** [comment]
No. The only charts we are actively maintaining are the OpenUnison (https://openunison.github.io/) and CI/CD proxy (https://cicd-proxy.github.io/) charts.

**Comment by JefeDavis** [comment]
@mlbiam y'all seem to be upgrading and maintaining the chart at the /deploy/charts folder? can y'all publish that?

**Comment by mlbiam** [comment]
We only publish and qa charts for OpenUnison. All other charts are community maintained.
